File: go/orm/sample/migrations/V0007.go
package migrations

import (
	"github.com/kosmassp/SimpleOrm/go/orm"
	"github.com/kosmassp/SimpleOrm/go/orm/sample/migrations/Table/User"
)

type V0007 struct{}

func (V0007) Compose(version *orm.VersionBuilder) {
	version.Apply(user.V0007_IndexDisplayName{})
}

File: php/samples/SimpleOrm.Sample/src/Migrations/V0007.php
<?php

declare(strict_types=1);

namespace SimpleOrm\Sample\Migrations;

use SimpleOrm\Migrations\MigrationVersion;
use SimpleOrm\Migrations\VersionBuilder;
use SimpleOrm\Sample\Migrations\Table\User\V0007_IndexDisplayName;

final class V0007 extends MigrationVersion
{
    public function compose(VersionBuilder $version): void
    {
        $version
            ->apply(V0007_IndexDisplayName::class);
    }
}
